build-tools/packages/build-tools/src/fluidBuild/buildGraph.ts

```diff
@@ -219,7 +219,10 @@ export class BuildGraph {
 				const child = this.buildPackages.get(name);
 				if (child) {
 					// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-					if (semver.satisfies(child.pkg.version, version!)) {
+					const satisfied =
+						version!.startsWith("workspace:") ||
```

Comment:
```suggestion
						version?.startsWith("workspace:") ||
```

Would this work?

Reply:
Well,, the next line also have a non-null-assertion already, so it should match?